packages/centreon-ui/src/Submenu/SubmenuHeader/index.tsx

```diff
@@ -16,19 +60,89 @@ const useStyles = makeStyles((theme) => ({
   bottom: {
     display: 'flex',
   },
+  countersTranslateToRight1: {
+    position: 'relative',
+    right: theme.spacing(1),
+    [theme.breakpoints.down(768)]: {
+      position: 'static',
+    },
+  },
+
+  countersTranslateToRight2: {
+    position: 'relative',
+    right: theme.spacing(4.3),
+    [theme.breakpoints.down(768)]: {
+      position: 'static',
+    },
+  },
+  link: {
+    textDecoration: 'none',
+  },
+  subMenuCounters: {
+    display: 'flex',
+    gap: theme.spacing(0.5),
+    [theme.breakpoints.down(768)]: {
+      display: 'grid',
+      gridTemplateColumns: 'auto auto',
+    },
+  },
+  subMenuRight: {
+    alignItem: 'flex-start',
+    display: 'flex',
+    flexDirection: 'column',
+    gap: theme.spacing(0.8),
+    justifyContent: 'space-between',
+    [theme.breakpoints.down(768)]: {
+      alignItems: 'center',
+      flexDirection: 'row',
+      gap: theme.spacing(0.1),
+    },
+  },
+  submenu: {
+    backgroundColor: theme.palette.common.black,
+    boxSizing: 'border-box',
+    display: 'none',
+    left: 0,
+    padding: `${theme.spacing(1)} ${theme.spacing(1)} 0 ${theme.spacing(1)}`,
+    position: 'absolute',
+    textAlign: 'left',
+    top: '100%',
+    width: theme.spacing(18),
+    zIndex: 99,
+  },
+  submenuDisplayed: {
+    display: 'block',
+  },
+
   top: {
-    alignItems: 'center',
     backgroundColor: theme.palette.common.black,
     display: 'flex',
-    flexWrap: 'wrap',
-    gridGap: theme.spacing(1),
-    padding: theme.spacing(1, 1, 1, 2),
     position: 'relative',
   },
+
+  wrapMiddleIcon: {
+    display: 'flex',
+  },
 }));
 
-const SubmenuHeader = ({ children, active, ...props }: Props): JSX.Element => {
+const SubmenuHeader = ({
+  active,
+  iconHeader,
+  counters,
+  iconToggleSubmenu,
+  submenuItems,
+  toggled,
+  hasPending,
+  ...props
+}: Props): JSX.Element => {
   const classes = useStyles();
+  const subMenuCounters =
+    // eslint-disable-next-line no-nested-ternary
+    iconHeader?.iconName === 'Services'
+      ? `${classes.subMenuCounters} ${classes.countersTranslateToRight1}`
+      : iconHeader?.iconName === 'Business Activities'
+      ? `${classes.subMenuCounters} ${classes.countersTranslateToRight2}`
+      : classes.subMenuCounters;
```

Comment:
Avoid nested ternary condition
```suggestion
    iconHeader?.iconName === 'Services'
      ? `${classes.subMenuCounters} ${classes.countersTranslateToRight1}`
      : iconHeader?.iconName === 'Business Activities'
      ? `${classes.subMenuCounters} ${classes.countersTranslateToRight2}`
      : classes.subMenuCounters;
```